### PR TITLE
Further increase frequence of the fill pk task in H

### PR DESCRIPTION
**Pull request by marcospri**
Start earlier, do the task every 10 minutes and for a bigger batch size

See performance on new relic:

https://one.newrelic.com/nr1-core/apm-features/transactions/MTM4NTI4M3xBUE18QVBQTElDQVRJT058MjI2ODg2MzE?account=1385283&duration=604800000&state=fe611650-259d-7fd0-114a-8a0f8f31d672

### CODE REVIEW

File: h_periodic/h_beat.py [hunk 1]
```diff
@@ -55,8 +55,8 @@
         "fill-annotations-pk": {
             "options": {"expires": 30},
             "task": "h.tasks.annotations.fill_pk_and_user_id",
-            "schedule": crontab(hour="*,9-12", minute="*/15"),
-            "kwargs": {"batch_size": 10000},
+            "schedule": crontab(hour="*,6-12", minute="*/10"),
+            "kwargs": {"batch_size": 15000},
```
Comment (marcospri) [inline h_periodic/h_beat.py:59]: Maybe more, 50k?

Comment (robertknight) [reply h_periodic/h_beat.py:59]: How long is this going to take to complete given the current DB size and growth rate?

Comment (marcospri) [reply h_periodic/h_beat.py:59]: With this rate over two months, new annotations are not that important, we can (and should) soon update the code so they don't need this, setting the pk and user_id in python when they are created.

File: h_periodic/h_beat.py [hunk 2]
```diff
@@ -55,8 +55,8 @@
         "fill-annotations-pk": {
             "options": {"expires": 30},
             "task": "h.tasks.annotations.fill_pk_and_user_id",
-            "schedule": crontab(hour="9-12", minute="*/15"),
-            "kwargs": {"batch_size": 10000},
+            "schedule": crontab(hour="5-12", minute="*/10"),
+            "kwargs": {"batch_size": 15000},
```
Comment (marcospri) [inline h_periodic/h_beat.py:59]: Some tasks have already gone over the time limit:

https://hypothesis.sentry.io/issues/4465586542/?referrer=slack&notification_uuid=adaafa01-d50f-4c25-b0ec-8cf3a50b4e10&alert_rule_id=7345746&alert_type=issue

I don't think we should increase, 4 min is plenty of time.

**Comment by marcospri** [comment]
Closed in favor of https://github.com/hypothesis/h-periodic/pull/351

**Pull request closed without merging**